### PR TITLE
test(e2e): add test to edge-function deploy

### PR DESCRIPTION
**Pull request by enicio**
Jira ENG-22229

### CODE REVIEW

File: test/e2e-test/flareact-function.test.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+import util = require('node:util');
```
Comment (augustollenz) [inline test/e2e-test/flareact-function.test.ts:1]: We need to merge this file with flareact.test.js to avoid code duplication and to avoid run the same test twice.

**Review by augustollenz — CHANGES_REQUESTED** [review]

File: test/e2e-test/flareact-function.test.ts (hunk 1 above)
Comment (augustollenz) [reply test/e2e-test/flareact-function.test.ts:1]: Actually this comment applies to whole file not to this specific line :)

File: test/e2e-test/flareact.test.ts [hunk 2]
```diff
@@ -69,46 +69,147 @@ describe('Create Flareact application', () => {
...
+                expect(filesOnS3BucketContainManifestFiles).to.be.true;
+            });
+        })
+
+        describe("Try to publish only function with wrong token and expect to fail", () => {
+            before(async () => {
+                await copy(path.join(localOutput, 'test', 'config-files', 'azion-flareact-with-wrong-token.json'),path.join(template,'azion.json'), { overwrite: true})
+            })
+
+            it('Try to publish only function with wrong token and expect to fail', async () => {
```
Comment (augustollenz) [inline test/e2e-test/flareact.test.ts:182]: Try to reprhase to avoid duplication between describe and it.

**Review by augustollenz — CHANGES_REQUESTED** [review]

**Review by augustollenz — APPROVED** [review]

**Review by MagnunAVFAzion — APPROVED** [review]
